### ISSUE TITLE
Exact steps to make it work on iOS.

### ISSUE THREAD

**Issue by coolvasanth** [aoberoi/cordova-plugin-opentokjs#27]
Hi All,

I have installed the opentokjs plugin also the iOSRTC plugin. On device ready I'm also calling 
`  cordova.plugins.iosrtc.registerGlobals();`. If I try to execute without iOSRTC plugin then I'm facing https://github.com/aoberoi/cordova-plugin-opentokjs/issues/13. If I install iOSRTC and try to call `  cordova.plugins.iosrtc.registerGlobals();` I'm getting some errors. 

All I need is a step by step guide to make it work on Ionic 3 and iOS 11. Or if there is any problem in supporting latest iOS 11, then I would request author to confirm it.

**Comment by hthetiot** [comment]
I dont think this plugin is maintained last update was Jun 25, 2015 it's like a century in WebRTC time.

You may want to check the original plugin instead, that is actively maintained.
- https://github.com/cordova-rtc/cordova-plugin-iosrtc
